paramak/parametric_components/port_cutters_rectangular.py

```diff
@@ -52,22 +58,39 @@ def __init__(
             **kwargs
         )
 
-        self.z_pos = z_pos
         self.height = height
         self.width = width
+        self.center_point = center_point
         self.fillet_radius = fillet_radius
-        self.add_fillet()
 
     def find_points(self):
+        if self.workplane[0] < self.workplane[1]:
+            parameter_1 = self.width
+            parameter_2 = self.height
```

Comment:
These lines aren't tested. Is there a way of adapting some of the tests?

Reply:
Thanks for pointing this out. Tests will be added